core/assertion.ts

```diff
@@ -213,6 +213,18 @@ export class AssertionContext implements ICommonContext {
     );
   }
 
+  public database(): string {
+    if (!this.assertion.proto.target.database) {
+      const message = `The database field in the target is undefined.`;
```

Comment:
inline the message?

also: `Warehouse does not support multiple databases`

Reply:
Thanks, updated and also inlined the `ref` message.